### PR TITLE
Controller folder

### PR DESCRIPTION
**Pull request by michaelryancaputo**

### CODE REVIEW

File: controller/index.js [hunk 1]
```diff
@@ -18,6 +18,6 @@ module.exports = yeoman.generators.Base.extend({
     this.nameSlug = this.name;
     this.nameSlugCC = _.camelCase(this.name);
     this.nameSlugKC = _.kebabCase(this.name);
-    this.template(this.templatePath('controller.js'), this.destinationPath('./' + this.nameSlugCC + 'Controller.js'));
```
Comment (thody) [inline controller/index.js]: The thinking here was actually that we not impose a particular directory structure on the user.

Pro: flexible directory structure; 
Con: onus is on the user to navigate to the appropriate controller before running the generator.

Thoughts?

**Comment by michaelryancaputo** [comment]
I'm cool with either, should remove the automatic controller folder creation if this was going to be the pattern we would follow.

**Comment by thody** [comment]
You may have an older version of the generator if you're still seeing a controller directory. The latest version doesn't generate a controller directory.

**Comment by michaelryancaputo** [comment]
bingo, thanks

**Pull request closed without merging**